GitPrompt.ps1

```diff
@@ -82,6 +82,9 @@ $global:GitPromptSettings = New-Object PSObject -Property @{
 
     AutoRefreshIndex                            = $true
 
+    # Valid values are "Full", "Compact", and "Minimal"
+    BranchBehindAndAheadDisplay                 = "Full"

```

Comment:
That's exactly why I left it default to Full and then added the example showing how to make it Compact.  I find that the staged files section takes way more space than the Full view, unless you're dealing with seriously out of sync branches.